scene/2d/polygon_2d.cpp

```diff
@@ -364,7 +369,31 @@ void Polygon2D::_notification(int p_what) {
 
 				arr[RS::ARRAY_INDEX] = index_array;
 
-				RS::get_singleton()->mesh_add_surface_from_arrays(mesh, RS::PRIMITIVE_TRIANGLES, arr, Array(), Dictionary(), RS::ARRAY_FLAG_USE_2D_VERTICES);
+				RS::SurfaceData sd;
+
+				if (skeleton_node) {
+					// Compute transform between mesh and skeleton for runtime AABB compute.
+
+					const Transform2D meshTransform = this->get_global_transform();
```

Comment:
```suggestion
					const Transform2D meshTransform = get_global_transform();
```
Unnecessary use of `this`